### PR TITLE
Hero component

### PR DESCRIPTION
**Pull request by portdeveloper**
### This PR:

-> Adds Space Mono font
-> Adds a Hero component with the image and the logo
-> Cleans up the main page to show hero and faq only

We can do some tweaks later on but for now this looks OK.

image

### CODE REVIEW

**Comment by portdeveloper** [comment]
Since this PR is relatively simple, I am adding the PrizeInfo component to it as well.

File: packages/nextjs/public/hero-image.png
Comment (carletex) [inline packages/nextjs/public/hero-image.png:1]: This image is 8 MBs haha. Let's reduce it (dimensions...and quality if needed). Ideally < 1 MB

Comment (portdeveloper) [reply packages/nextjs/public/hero-image.png:1]: I have managed to get it down to 277 kb without losing much quality, I hope that works

**Comment by technophile-04** [comment]
Great work @portdeveloper! look great!! Just pushed a minor commit keeping `font-spaceMono` at top of app tree so that wherever we use text it should by default `spaceMono` and we don't need to add that className for all text

**Pull request merged**